make_ggml.py

```diff
@@ -0,0 +1,45 @@
+import argparse
+import os
+import subprocess
+
+def main(model, outbase, outdir, quants):
+    ggml_version = "v3"
+
+    if not os.path.isdir(model):
+        raise Exception(f"Could not find model dir at {model}")
+
+    if not os.path.isfile(f"{model}/config.json"):
+        raise Exception(f"Could not find config.json in {model}")
+
+    os.makedirs(outdir, exist_ok=True)
+
+    print("Building llama.cpp")
+    subprocess.run(f"make clean && make", shell=True, check=True)
+    # subprocess.run(f"cd {llamabase} && make clean && LLAMA_METAL=1 make", shell=True, check=True)
+
+    fp16 = f"{outdir}/{outbase}.ggmlv2.fp16.bin"
```

Comment:
I hardcoded this to ggmlv2 back in the day - it should use ggml_version in case the user wants to keep it